src/redist/targets/GenerateLayout.targets

```diff
@@ -343,6 +337,15 @@
       </BundledInstallerComponent>
     </ItemGroup>
 
+    <ItemGroup>
+      <BundledLayoutComponent Include="CombinedSharedHostAndFrameworkArchive">
+        <BaseUrl>$(CoreSetupRootUrl)$(CoreSetupBlobVersion)</BaseUrl>
+        <BaseUrl Condition=" '$(DotNetBuildFromSource)' == 'true' AND '$(DotNetBuildFromSourceFlavor)' == 'Product' ">$(CoreSetupRootUrl)</BaseUrl>
+        <DownloadFileName>$(CombinedFrameworkHostArchiveFileName)</DownloadFileName>
+        <RelativeLayoutPath></RelativeLayoutPath>
+      </BundledLayoutComponent>
+    </ItemGroup>
+
```

Comment:
We can undo moving this down to the bottom of the list now that we're specifically requesting packaging subdirectories.